src/components/StepIndicator/__tests__/StepIndicator.spec.js

```diff
@@ -4,40 +4,54 @@
  */
 
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
 
-import { Step, StepIndicator } from '../';
+import { Step, StepIndicator } from '../../..';
```

Comment:
It looks like we ultimately spread the `Step` props into `ProgressStep`, but I believe we'll need to account for that logic, as well as potentially the `renderLabel` prop

Reply:
Thank you -- just added a new file for `Step` tests in particular 👍